.github/workflows/zephyr.yml

```diff
@@ -18,14 +18,11 @@ jobs:
         # many lines. Pay attention to COMMAS.
         IPC_platforms: [
           # - IPC3 default
-          apl cnl,
-          icl jsl,
-          tgl tgl-h,
           imx8 imx8x imx8m,
           # - IPC4 default
           mtl,
           # Very few IPC3 platforms support IPC4 too.
-          -i IPC4 tgl,
+          -i IPC4 tgl
```

Comment:
do we need tgl-h here too for ipc4/Zephyr @kv2019i @mengdonglin do we have tgl-h dev boards ?

Reply:
@lgirdwood we can add tgl-h later maybe?
This PR is blocking some other PRs, can we integrate it?